src/dk/aau/cs/ds306e18/tournament/rlbot/configuration/ConfigFile.java

```diff
@@ -0,0 +1,280 @@
+package dk.aau.cs.ds306e18.tournament.rlbot.configuration;
+
+import java.io.*;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Set;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+/**
+ * Abstraction of a '.cfg'-file.
+ */
+public class ConfigFile {
+
+    // Can parse files like this: https://docs.python.org/3/library/configparser.html#supported-ini-file-structure
+    // Inspired by: https://stackoverflow.com/a/15638381
+
+    private final Pattern sectionPattern = Pattern.compile("\\[([^\\[\\]]*)\\]\\s*");
```

Comment:
A short description for each pattern could be a nice addition, such that most people will be able to decipher the syntax.

Reply:
Added! (11f2e2e7fbd3275427)